src/sentry/search/events/builder.py

```diff
@@ -852,12 +853,17 @@ def custom_measurement_map(self) -> List[MetricMeta]:
 
         from sentry.snuba.metrics.datasource import get_custom_measurements
 
-        result: List[MetricMeta] = get_custom_measurements(
-            project_ids=self.params["project_id"],
-            organization_id=self.organization_id,
-            start=datetime.today() - timedelta(days=90),
-            end=datetime.today(),
-        )
+        try:
```

Comment:
Should the metrics queries be behind a feature flag?

Reply:
Good call, updated in ba5e966cd9254dad8f959f7e8fc127041a4d312e